src/App.css

```diff
@@ -1,14 +1,27 @@
+/* Repaired text colour issue using a tag */
+
 .App {
   text-align: center;
-  color: rgb(180, 110, 4);
+  color: black;
+}
+/* changed the text colour -> blk header */
+
+
+a {
+  color: rgb(177, 109, 6);
 }
+/* added a tag to counter the effect of the colour change */
 
 .container {
-  display: flex;
+  /* display: flex;
```

Comment:
Please remove commented code - old stuff can be reviewed on the Git history

Reply:
Done. Thank you